sdk/iot_deviceupdate/src/client.rs

```diff
@@ -7,7 +7,7 @@ use const_format::formatcp;
 use serde::de::DeserializeOwned;
 use std::sync::Arc;
 
-pub(crate) const API_VERSION: &str = "2021-06-01-preview";
+pub(crate) const API_VERSION: &str = "2022-10-01";
```

Comment:
should we update the iot_deviceupdate paackage version regarding the changes?
https://github.com/JanZachmann/azure-sdk-for-rust/blob/8ff7922e284ec70d327c46650d3cb3379f08c0ec/sdk/iot_deviceupdate/Cargo.toml#L3